admin/acceptance_tests/layers_wms_test.py

```diff
@@ -380,11 +383,25 @@ class TestLayerWMSSelenium():
 
     _prefix = '/layers_wms'
 
-    def test_index(self, selenium, selenium_app):
+    def test_index(self, selenium, selenium_app, layer_wms_test_data):
         selenium.get(selenium_app + self._prefix)
 
+        layer = layer_wms_test_data['layers'][5]
         index_page = IndexPage(selenium)
         index_page.select_language('en')
         index_page.check_pagination_info('Showing 1 to 25 of 26 rows', 10)
         index_page.select_page_size(10)
         index_page.check_pagination_info('Showing 1 to 10 of 26 rows', 10)
+
+        WebDriverWait(selenium, 10).until(
+            lambda driver: driver.execute_script(
+                'return (window.jQuery != undefined && jQuery.active == 0)'))
+
+        el = index_page.find_element(
+            By.XPATH,
+            '//td[contains(text(),"{}")]'.format(layer.geo_table),
+            timeout=5)
+        action_chains = ActionChains(selenium)
+        # following event implies use of chrome selenium web driver instead of gecko (Firefox)
+        action_chains.move_to_element(el).double_click().perform()
+        assert selenium.current_url.endswith('/layers_wms/{}'.format(layer.id))
```

Comment:
Sorry to be late, but is it possible to factorize this in IndexPage please.

Reply:
You are not late.